packages/client/lib/AuthorizationCodeClient.ts

```diff
@@ -17,18 +21,28 @@ import Debug from 'debug';
 
 const debug = Debug('sphereon:oid4vci');
 
+function filterSupportedCredentials(
+  credentialOffer: CredentialOfferPayloadV1_0_13,
+  credentialsSupported?: Record<string, CredentialConfigurationSupportedV1_0_13>,
+): CredentialConfigurationSupportedV1_0_13[] {
+  if (!credentialOffer.credential_configuration_ids || !credentialsSupported) {
+    return [];
```

Comment:
Are you sure this is an or. Looks more like it should be an and. If not, then this code is now tied to v13 by the looks of it. Probably wise to make a comment about that.
edit: I see that it gets called when v13 is detected, so please add the comment